torch/csrc/jit/python/script_init.cpp

```diff
@@ -710,6 +710,22 @@ void extra_files_to_python(const ExtraFilesMap& m, const py::dict& pydict) {
   }
 }
 
+void compilationUnit_py_define(
```

Comment:
```suggestion
void pyCompilationUnitDefine(
```
Based on the name of another similar function `pyIValueDeepcopy` in this file.

Reply:
Right. Always take the closest precedent.